using-mobile-extensions/mobile-core/identity/identity-api-reference.md

```diff
@@ -797,28 +900,32 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 {% tab title="React Native" %}
 #### JavaScript
 
-### setAdvertisingIdentifier
+### setAdvertisingIdentifier<a id="setAdvertisingIdentifier-js"></a>
 
 ```jsx
 ACPCore.setAdvertisingIdentifier("adID");
 ```
+o *adID* is a String that provides developers with a simple, standard system to continue to track the Ads through their apps	
```

Comment:
Reword to:
"* *adID* is a string that provides developers with a simple, standard system to track the ads through their apps.